### ISSUE TITLE
Laravel 9?

### ISSUE THREAD

**Issue by oriceon** [anlutro/laravel-settings#157]
```
λ composer require anlutro/l4-settings
Using version ^1.0 for anlutro/l4-settings
./composer.json has been updated
Running composer update anlutro/l4-settings
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires anlutro/l4-settings ^1.0 -> satisfiable by anlutro/l4-settings[v1.0.0].
    - anlutro/l4-settings v1.0.0 requires illuminate/support >=4.2 <9.0 -> found illuminate/support[v4.2.0-BETA1, ..., 4.2.x-dev, v5.0.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.

You can also try re-running composer require with an explicit version constraint, e.g. "composer require anlutro/l4-settings:*" to figure out if any version is installable, or "composer require anlutro/l4-settings:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by bweston92** [comment]
Hey @oriceon do you mind submitting a PR and I would be happy to review.

**Comment by bweston92** [comment]
Closed by #158
